ctest-next/tests/input/all_items.rs

```diff
@@ -0,0 +1,23 @@
+use std::os::raw::c_void;
+
+mod level1 {
```

Comment:
Huh... we're not checking module visibility are we. Could you add it here if it's trivial?

If not don't worry about it, can be fixed later.

Reply:
Alright, this is fine as-is then